src/index.html

```diff
@@ -6,13 +6,63 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
+    <title>Product 8768767</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/style.scss"
```

Comment:
The stylesheet link is pointing to a SCSS file (`style.scss`). SCSS files need to be compiled to CSS before they can be used in an HTML document. Ensure that you compile your SCSS to CSS and link the resulting CSS file instead.